### ISSUE TITLE
[FeatureRequest] Set-VMFirmware [-BootOrder <VMComponentObject []>]

### ISSUE THREAD

**Issue by hmaster20** [taliesins/terraform-provider-hyperv#67]
Hello!
Please implement the feature of changing the boot priority of the virtual machine.
For PowerShell, this is used to launch with a key:
[Set-VMFirmware [-BootOrder <VMComponentObject []>]
](https://docs.microsoft.com/en-us/powershell/module/hyper-v/set-vmfirmware?view=windowsserver2019-ps)

For example, setting the boot priority from the network interface
```powershell
$vmNetworkAdapter = Get-VMNetworkAdapter -VMName WebServer
$vmHardDiskDrive = Get-VMHardDiskDrive -VMName WebServer
Set-VMFirmware -VMName WebServer -BootOrder $vmNetworkAdapter, $vmHardDiskDrive
```

For example setting boot priority from disk
```powershell
$vmNetworkAdapter = Get-VMNetworkAdapter -VMName WebServer
$vmHardDiskDrive = Get-VMHardDiskDrive -VMName WebServer
Set-VMFirmware -VMName WebServer -BootOrder $vmHardDiskDrive, $vmNetworkAdapter
```

What is it for ?
If a machine is created in the off state, then the first priority is network booting through the PXE
If you turn on such a machine, then it will not boot.
To boot, you must manually change the priority to disk.

Please add the load priority selection feature to your plugin!

**Comment by taliesins** [comment]
This will be resolved by #122

**Comment by taliesins** [comment]
This feature has not been implemented
